src/Engine/Util/Memory.cpp

```diff
@@ -2,7 +2,7 @@
 
 #include "ThirdParty/CoherentLabs_rpmalloc/rpmalloc.h"
```

Comment:
Instead of ifdef'ing the rpmalloc source files, could you simply ifdef this include? I'd rather not modify third party source if possible

Reply:
I will be able to take out the ifdef in rpmalloc.h but am unable to find a way that allows taking out the ifdef in rpmalloc.cpp

Reply:
You're saying simply compiling rpmalloc.cpp causes the problem?

Reply:
If I take the ifdefs out in the cpp files I get the following

rpmalloc 366: error: thread-local storage is not supported for the current target
RPMALLOC_DEFINE_THREAD_LOCAL(uint32_t, _memory_preferred_heap)

Is there a way to not include the file during compilation?